spec/salesforce_id/safe_spec.rb

```diff
@@ -34,6 +34,16 @@
     expect{subject.new(nil)}.to raise_error ArgumentError
   end
 
+  describe "#initialize" do
+    it "raises an ArgumentError if the parameter isn't valid" do
+      begin
+        SalesforceId::Safe.new(error_short_id)
```

Comment:
What about

```ruby
expect {
  SalesforceId::Safe.new(error_short_id)
}.to raise_error ArgumentError, "Salesforce ID (#{error_short_id}) not valid"
```

In this way you ensure that the error is actually raised, otherwise test might not fail if nothing was raised (false positive)